packages/inputs-minimal/src/radio_group/index.jsx

```diff
@@ -1,41 +1,59 @@
 import PropTypes from 'prop-types';
-import clsx from 'clsx';
 import Radio from '../radio';
 import styles from './index.module.scss';
+import { Children } from 'react';
+
+export const RadioGroupHorizontalLayout = ({ children }) => (
+  <div className={styles.horizontalLayout}>
+    {Children.map(children, (child) => (
+      <div className={styles.item}>{child}</div>
+    ))}
+  </div>
+);
+
+export const RadioGroupVerticalLayout = ({ children }) => (
+  <div className={styles.verticalLayout}>
+    {Children.map(children, (child) => (
+      <div className={styles.item}>{child}</div>
+    ))}
+  </div>
+);
 
 const RadioGroup = ({
   onChange,
-  className,
   items,
   name,
   value,
+  Layout = RadioGroupHorizontalLayout,
```

Comment:
In what case would we need to pass a different layout here aside from `horizontal` or `vertical`?

I propose a different prop, `display` and the value will be `inline` or `block`.  Or we do  `layout` as prop name and acceptable values `horizontal` and `vertical`.